src/test/java/no/sikt/sws/WorkspaceStripperTest.java

```diff
@@ -91,61 +89,76 @@ Stream<DynamicTest> testPrefixBodyFactory() {
+
+        new TestCaseLoader(filename)
+            .getTestCases().forEach(this::assertBodyPrefixAlias);
+    }
+
+
+
     void assertResponseStripping(TestCaseSws testCase) {
         var expectedResponse = testCase.getResponseStripped();
-        var resultResponse = WorkspaceStripper.remove(testCase.getResponse(), WORKSPACEPREFIX);
+        var openSearchResponse = testCase.getResponse();
+        var resultResponse = WorkspaceStripper.remove(openSearchResponse, WORKSPACEPREFIX);
 
         assertEquals(expectedResponse,resultResponse);
 
         logger.info(testCase.getRequestOpensearch().getMethod() + "->" + testCase.getRequestOpensearch().getUrl());
+        logger.info(resultResponse);
     }
 
     void assertUrlPrefixing(TestCaseSws testCase) {
         var gatewayUrl = testCase.getRequestGateway().getUrl();
         var expectedUrl = testCase.getRequestOpensearch().getUrl();
         var resultUrl = WorkspaceStripper.prefixUrl(gatewayUrl, WORKSPACEPREFIX);
 
+        logger.info(gatewayUrl + "->" + expectedUrl);
+
         assertEquals(expectedUrl,resultUrl);
 
-        logger.info(gatewayUrl + "->" + expectedUrl);
     }
 
-    void assertBodyPrefixing(TestCaseSws testCase) {
-        var indexName = testCase.getIndexName();
-
-        if (indexName != null) {
-            //var expectedBody = testCase.getRequestOpensearch().getBulkBody();
-            //var gatewayBody = testCase.getRequestGateway().getBulkBody();
-            //var resultBody = WorkspaceStripper.prefixBody(gatewayBody, WORKSPACEPREFIX);
-            //assertEquals(expectedBody,resultBody);
+    void assertBodyPrefixAlias(TestCaseSws testCase) {
+        var expectedBody = testCase.getRequestOpensearch().getBody();
+        var gatewayBody = testCase.getRequestGateway().getBody();
+        var resultBody = WorkspaceStripper.prefixAliasBody(gatewayBody, WORKSPACEPREFIX);
 
-            //} else {
-            var expectedBody = testCase.getRequestOpensearch().getBody();
-            var gatewayBody = testCase.getRequestGateway().getBody();
-            var resultBody = WorkspaceStripper.prefixBody(gatewayBody, WORKSPACEPREFIX, indexName);
-            assertEquals(expectedBody,resultBody);
-        }
+        assertEquals(expectedBody,resultBody);
+        logger.info(resultBody);
 
-        logger.info(testCase.getRequestOpensearch().getMethod() + "->" + testCase.getRequestOpensearch().getUrl());
     }
 
-    void assertBulkBodyPrefixing(TestCaseSws testCase) {
-        var gatewayBody = testCase.getRequestGateway().getBulkBody();
+    void assertBodyPrefixing(TestCaseSws testCase) {
+        var indexName = testCase.getIndexName();
         var expectedBody = testCase.getRequestOpensearch().getBody();
-        var resultBody = WorkspaceStripper.prefixBody(gatewayBody, WORKSPACEPREFIX);
 
-        assertEquals(expectedBody,resultBody);
+        if (indexName == null) {
+            logger.info("no index name, presuming bulk inserts");
```

Comment:
instead of checking indexName == null here, you could iterate though request-bulk.json and for each run a assertBodyBulkPrefixing() and similar for the other test-types you create.